### DISCUSSION TITLE
Adding the ability to play specific or a group of determined sounds on specific animation frames.

### DISCUSSION THREAD

**Discussion by Akis-02** [OpenXRay/xray-16#1513] [Ideas]
Well the tittle explains it mostly, this idea mostly comes from the source engine. Basically you can add a specific or a predetermined group of sounds to play on specific animation frames. The specified sound group picks one sound and  plays it randomly whenever the call for sound is asked.

**Comment by Akis-02** [comment]
This removes the need of adding long audio tracks and also give the potential of more variety to stuff like gun reloads or general Foley

**Comment by Xottab-DUTY** [comment]
This is a feature request, not a discussion!

**Reply by Akis-02** [reply]
In the ideas topic

**Comment by Xottab-DUTY** [comment]
Superseded by #1519.
